File: ex03/ex03_03/ex03_03.js
import promptSync from 'prompt-sync';
const prompt = promptSync();

function main() {
  let x = 0;
  const low = 11;
  const up = 20;
  let condition1 = false;
  let condition2 = false;
  let condition3 = false;

  x = parseInt(prompt("Enter x: "));

  condition1 = (x >= low) && (x <= up);
  condition2 = (x < low) || (x > up);
  condition3 = !(x >= up);

  console.log(String(x) + " is within [" + String(low) + ", " + String(up) + "]: " + condition1);
  console.log(String(x) + " is outside [" + String(low) + ", " + String(up) + "]: " + condition2);
  console.log(String(x) + " is less than " + String(up) + ": " + condition3);
 
  return 0;
}
main();

File: ex03/ex03_03/ex03_03.py
def main():
  x = 0
  low = 11
  up = 20
  condition1 = False
  condition2 = False
  condition3 = False

  x = int(input("Enter x: "))

  condition1 = (x >= low) and (x <= up)
  condition2 = (x < low) or (x > up)
  condition3 = not (x >= up)

  print(str(x) + " is within [" + str(low) + ", " + str(up) + "]: " + str(condition1))
  print(str(x) + " is outside [" + str(low) + ", " + str(up) + "]: " + str(condition2))
  print(str(x) + " is less than " + str(up) + ": " + str(condition3))
  
  return 0

if __name__ == "__main__":
  main()